C-git-commands.asc

```diff
@@ -62,7 +62,7 @@ Accompanying the configuration instructions in <<ch01-getting-started#_editor>>,
 |Scratch (Linux)|`git config --global core.editor "scratch-text-editor"`
 |Sublime Text (macOS) |`git config --global core.editor "/Applications/Sublime\ Text.app/Contents/SharedSupport/bin/subl --new-window --wait"`
 |Sublime Text (Windows 64-bit) |`git config --global core.editor "'C:\Program Files\Sublime Text 3\sublime_text.exe' -w"` (Also see note below)
-|TextEdit (macOS)|`git config --global --add core.editor "open -W -n"`
+|TextEdit (macOS)|`git config --global core.editor "open -W -n"`
```

Comment:
```suggestion
|TextEdit (macOS)|`git config --global core.editor "open --wait-apps --new -e"`
```

This suggestion implements @ben's `core.editor` config that he tried on Catalina.